Add Autoimmune Disease Machine Learning Challenge #1 and #2

Caceresenzo opened the pull request.

hcarlens commented:
Thanks @Caceresenzo! A few questions:
1. The page mentions a $50k prize pool; how are those prizes distributed across leaderboards in each of the "crunches"? 
2. Can participants choose to take part in one of the three "crunches" and not the others? I.e., could it make sense to show this as three competitions with different deadlines on ML Contests?

Caceresenzo commented:
1. You are right, it is spread over the 3 competitions ([docs](https://docs.crunchdao.com/competitions/competitions/broad-institute-autoimmune-disease#prizes)).
  Should I change it in the JSON?
  Here is the distribution:
    - 1st competition: $12,000
    - 2nd competition: $12,000
    - 3rd competition: $26,000
2. In fact, they can only enter one competition and not the others. Since they have different opening times (but all the same deadline), I indeed wanted to add them as 3 separate competition.

> [!NOTE]
> "Crunch" is the marketing term for competitions.
> Since it is hard to understand if you are not familiar with our platform, I use "competition" which is more understandable for external people.

Caceresenzo commented:
I added the 2nd competition, and updated the prizes.
I cannot add the 3rd for now as it only lunch on December 9th and we do not allow pre-registration.

Rewards will be distributed to the top 10. ([reward scheme](https://docs.crunchdao.com/competitions/competitions/broad-institute-autoimmune-disease#prizes))
